tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -617,10 +617,10 @@ $ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
 # To build with GPU support:
 $ bazel build -c opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
 
-$ bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
+$ sudo bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
```

Comment:
> @@ -617,10 +617,10 @@ $ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
>  # To build with GPU support:
>  $ bazel build -c opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
>
> -$ bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
> +$ sudo bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg

In case of this, "/tmp/tensorflo_pkg" is as same as usage of other OS
platform like Ubuntu. but OS X environment   "/tmp" location of file system
is need to sudo permission. current package location occur to OS
dependency. I think that build and install from source code is not so good
specific location. If it is possible to change other description from
"/tmp/tensorflow_pkg" to other location, it is more flexible way to build
and installation.

On Tue, Apr 26, 2016 at 8:15 AM, Martin Wicke <notifications@github.com>
wrote:

> In tensorflow/g3doc/get_started/os_setup.md
> <https://github.com/tensorflow/tensorflow/pull/2095#discussion_r61006165>:
>
> > @@ -617,10 +617,10 @@ $ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
> >  # To build with GPU support:
> >  $ bazel build -c opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
> >
> > -$ bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
> > +$ sudo bazel-bin/tensorflow/tools/pip_package/build_pip_package /tmp/tensorflow_pkg
>
> You really shouldn't need sudo for this one, unless your /tmp folder isn't
> writable for your user, or wherever you downloaded tensorflow to isn't.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/2095/files/7d1417ce568bb5d28b215d7272549751cd599c63#r61006165>
>